lib/CustomGroupsBackend.php

```
@@ -204,4 +204,19 @@ private function formatGroupId($uri) {
 	public function isVisibleForScope($scope) {
 		return ($scope === 'sharing');
 	}
+
+	/**
+	 * Delete group
+	 *
+	 * @param string $gid group id
+	 */
+	public function deleteGroup($gid) {
```

I don't have the interface at hand, but this method should handle error cases somehow. What if the group can't be deleted? It should either throw a specific exception or return true / false to indicate success or failure.